server/routes/record-attendance/handlers/editAttendance.ts

```diff
@@ -0,0 +1,130 @@
+import { Request, Response } from 'express'
+import { areIntervalsOverlapping, parse } from 'date-fns'
+import { Expose } from 'class-transformer'
+import { IsIn } from 'class-validator'
+import ActivitiesService from '../../../services/activitiesService'
+import PrisonService from '../../../services/prisonService'
+import { toDate } from '../../../utils/utils'
+import { ScheduledActivity, ScheduledEvent } from '../../../@types/activitiesAPI/types'
+
+enum EditAttendanceOptions {
+  YES = 'yes',
+  NO = 'no',
+  REMOVE = 'remove',
+}
+
+export class EditAttendance {
+  @Expose()
+  @IsIn(Object.values(EditAttendanceOptions), { message: 'Select an attendance option' })
+  attendanceOption: string
+}
+
+export default class EditAttendanceRoutes {
+  constructor(private readonly activitiesService: ActivitiesService, private readonly prisonService: PrisonService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const { id } = req.params
+    const { attendanceId } = req.params
+
+    const instance = await this.activitiesService.getScheduledActivity(+id, user).then(i => ({
+      ...i,
+    }))
```

Comment:
Same question as in attendanceDetails.ts

Reply:
removed